tauri-app/src-tauri/src/traymenu.rs

```diff
@@ -0,0 +1,65 @@
+use tauri::{
+    menu::{CheckMenuItemBuilder, MenuBuilder, MenuItemBuilder, PredefinedMenuItem},
+    tray::{ClickType, TrayIconBuilder},
+    Icon, Manager
+};
+use tauri_plugin_dialog::DialogExt;
+
+use crate::window::open_or_create_window;
+
+pub fn setup_tray_menu(app: &mut tauri::App) -> core::result::Result<(), Box<dyn std::error::Error>> {
+    // Tray menu
+    let close_btn = MenuItemBuilder::with_id("close", "Exit").build(app);
+    let open_btn = MenuItemBuilder::with_id("open", "Open").build(app);
+
+    let spn = CheckMenuItemBuilder::with_id("spn", "SPN").build(app);
+
+    let menu = MenuBuilder::new(app)
+        .items(&[
```

Comment:
acutally load the status of the SPN etc and display the check-menu-items accordingly